src/mds/CInode.cc

```diff
@@ -70,6 +70,12 @@ void CInodeCommitOperation::update(ObjectOperation &op, inode_backtrace_t &bt) {
   bufferlist layout_bl;
   encode(_layout, layout_bl, _features);
   op.setxattr("layout", layout_bl);
+
+  if (!symlink.empty()) {
+    bufferlist symlink_bl;
+    encode(symlink, symlink_bl);
+    op.setxattr("symlink", symlink_bl);
```

Comment:
It's a little confused here, which `symlink` you want to use here ? I see there also has one `symlink` member in the `CInodeCommitOperation` class. And you are also trying to set `op.symlink` in linke #1407 bellow.

Reply:
Sorry, I missed reading it. It's `CInodeCommitOperations` class not `CInodeCommitOperation`.

Please ignore this.